### PR TITLE
MS-178 refactor(code): code refactoring after mentor's check

### PR DESCRIPTION
**Pull request by ifbfirst**
1. Issue: https://github.com/nadyavalin/moon-store/issues/178
2. Task: -
3. Screenshot:
-
4. Branch refactor/MS-178

### CODE REVIEW

File: src/pages/basket/basketHandler.ts [hunk 1]
```diff
@@ -70,8 +42,13 @@ export function decreaseQuantityProduct(
       },
     ])?.then((response) => {
       if (response.statusCode === 200) {
+        if (quantity === 1) {
+          btn.classList.add("disabled");
+        } else {
+          btn.classList.remove("disabled");
+        }
```
Comment (arcanar7) [inline src/pages/basket/basketHandler.ts]: вместо if/else - `btn.classList.toggle("disabled", quantity === 1);`

Comment (ifbfirst) [reply src/pages/basket/basketHandler.ts]: Super!Done.

**Review by arcanar7 — APPROVED** [review]

**Review by nadyavalin — APPROVED** [review]
Ok!

**Review by raenlin — APPROVED** [review]
agree

**Pull request merged**